src/main/scala/org/apache/spark/sql/TiStrategy.scala

```diff
@@ -200,10 +203,46 @@ class TiStrategy(context: SQLContext) extends Strategy with Logging {
         originalAggExpr.isDistinct,
         NamedExpression.newExprId)
 
+    /**
+      * Is expression allowed to be pushed down
+      * @param expression the expression to examine
+      * @return whether expression can be pushed down
+      */
+    def allowExpPushDown(expression: Expression): Boolean = {
```

Comment:
We have test utils called isSupportedFilter
Please move it there.

Reply:
Done.